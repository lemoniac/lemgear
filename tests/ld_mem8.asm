ld BC, heap
ld A, $ce
ld (BC), A
ld A, 0
ld A, (heap)
ld D, A

ld A, $ec
ld (heap), A
ld A, 0
ld A, (BC)
ld E, A

ld HL, heap
ld (HL), $aa
ld A, (heap)

halt

	nop
	nop
heap:
	nop
	nop

