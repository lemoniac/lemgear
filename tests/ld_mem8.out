AF : AA00
DE : CEEC
